AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -40,22 +41,9 @@ - (void)viewDidLoad
 
   // This is the simpliest way to print Browser word using custom bold font.
   CGFloat fontSize = self.adblockBrowserLabel.font.pointSize;
-  UIFont *lightFont = [Appearence defaultLightFontOfSize:fontSize];
```

Comment:
These changes are for translations as well, right? Like I said below, we should focus on one thing in this pull request - issue 3093.